### DISCUSSION TITLE
Change view

### DISCUSSION THREAD

**Discussion by alexherodotou** [iSoron/uhabits#1963] [Help & FAQ]
Is there a way to change the number of days shown on the main screen? My phone screen is 6.5 inches and I can still only see 4 days, but I'd really like to see 5. Is there anything I can do?

**Comment by royappa** [comment]
I came looking for this same thing, because the github screenshots show 5 days but mine only shows 4. 
I found out that if you turn the camera sideways then you see 10! Which is fabulous, but you may have to then scroll to see all habits.
But I keep my display orientation locked so I only found this out by accident.

So it looks like the program automatically sets the number of days based on the available width. Like you, I would like to see more days in vertical mode, and the habit names can wrap or you are forced to shorten them. I can live with very short habit names.

Thanks for a great app.
